lona/events/view_event.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from typing_extensions import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from lona.view import LonaView
+
+
+@dataclass
+class ViewEvent:
+    name: str
+    view_classes: list[type['LonaView']]
+    data: dict | None = None
```

Comment:
nit: Maybe we can simplify it? `None` to `dict` conversion happens inside `fire_view_event` method.
```suggestion
    data: dict
```

Reply:
Good point. I will change the type to ``dict`` only.